### ISSUE TITLE
New Gender Option needed for Awards

### ISSUE THREAD

**Issue by millsjdm** [barberscore/barberscore-api#527]
The current gender choices for awards are Men's, Women's & Mixed. Since some awards will be available to any group category, we need an additional choice labeled as "Open".

**Comment by dbinetti** [comment]
TL;DR This is implemented in be51cd3e by considering "Open" as null.

This can't be done because that's now how the field works.  Think of this less as a category and more as a matching field; meaning, it matches the groups of the selected gender.  Since "open" is not itself a category like M/F/X then it can't match.  Instead, I've made it nullable, which accomplishes the same goal.

The way to think about it is as follows:  choose a Gender when the award is *restricted* to that gender and only that gender.  If left null, then the gender of the groups is irrelevant for the purposes of determining eligibility.  Make sense?

**Comment by millsjdm** [comment]
That works! Null = "open" - got it!
Can we add a filter on the backend to search for null?

**Comment by CLBuechler2324** [comment]
Gee, that was my question...

**Comment by dbinetti** [comment]
it's not built in, but I'm sure I can.

in the near term, the way to do it is to choose "all" then sort by Gender

**Comment by CLBuechler2324** [comment]
I'm fine with not choosing a gender when creating award.  However, doesn't appear to be way o remove existing gender.

**Comment by dbinetti** [comment]
You pick the option with the '---------' at the top.  It wasn't available when you checked because the code hadn't deployed.

**Comment by CLBuechler2324** [comment]
I sent a thank you 20 minutes ago. 🤓  Have first open/null awards already.

**Comment by dbinetti** [comment]
ha!

**Comment by dbinetti** [comment]
Just so we're all clear:  given the changes you made, the CSD Quartet
Championship will now be determined by all groups in the Session,
regardless of gender.  This is what you want, correct?

**Comment by CLBuechler2324** [comment]
Well, explicitly it should be the groups that are included in the award. Some groups may elect not to compete for Open championship, so technically it would not be all groups in the session.  Does that mean Null is an issue?

I discussed with Carter as CSD DRCJ. They will have four quartet & chorus champions - Open, Men, Women, Mixed.

**Comment by dbinetti** [comment]
You are correct, only those who have selected it will be eligible.  I
should have been clearer.
